extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -129,15 +152,36 @@
                 "        if (headers):\n",
                 "            for header in headers:\n",
                 "                title = header[10:].rstrip()\n",
-                "                contentFolders.append(title.lower())\n",
-                "                filtered = list(filter(lambda x: (\"%s%s%s\" % (os.path.sep, title.lower(), os.path.sep)) in x, urls))\n",
+                "                filtered = list(filter(lambda x: (\"%s%s%s\" % (os.path.sep, title.lower(), os.path.sep)) in x.lower(), urls))\n",
                 "                index = urls.index(filtered[len(filtered)-1])\n",
+                "                markdownFilePath = getMarkdownFile(filtered[len(filtered)-1])\n",
```

Comment:
In python you should be able to just do `filtered[-1]` to access the last item in a list

Reply:
(there's a whole bunch of places where this is being done - that might be worth doing in a separate PR though to keep this one a bit cleaner)

Reply:
didn't check if that works, will do it in a seperate PR. Thanks.